pkg/controller/component/lorry_utils.go

```diff
@@ -477,8 +475,10 @@ func getActionCommandsWithExecImageOrContainerName(synthesizeComp *SynthesizedCo
 		// "reconfigure":                synthesizeComp.LifecycleActions.Reconfigure,
 	}
 
-	if synthesizeComp.LifecycleActions.RoleProbe != nil {
-		actions[constant.RoleProbeAction] = &synthesizeComp.LifecycleActions.RoleProbe.LifecycleActionHandler
+	if synthesizeComp.LifecycleActions.RoleProbe != nil && synthesizeComp.LifecycleActions.RoleProbe.Exec != nil {
+		actions[constant.RoleProbeAction] = &appsv1alpha1.LifecycleActionHandler{
+			CustomHandler: &synthesizeComp.LifecycleActions.RoleProbe.Action,
+		}
```

Comment:
need set BuiltInHandler? 
```
if synthesizeComp.LifecycleActions.RoleProbe != nil && synthesizeComp.LifecycleActions.RoleProbe. builtinHandler != nil {
		actions[constant.RoleProbeAction] = &appsv1alpha1.LifecycleActionHandler{
			BuiltInHandler: &synthesizeComp.LifecycleActions.RoleProbe.builtinHandler,
		}
}
```

Reply:
The purpose here is to obtain contexts about the custom action, without considering the built-in handler.